kahuna/public/js/search/index.js

```diff
@@ -68,6 +74,33 @@ search.config(['$stateProvider',
                 // *don't pollute the $stateParams
                 delete $stateParams.isDeepStateRedirect;
                 return isDeepStateRedirect;
+            }],
+            selection: ['orderedSetFactory', function(orderedSetFactory) {
+                return orderedSetFactory();
+            }],
+            // equivalent of `ctrl.imagesAll`
+            results: ['listFactory', function(listFactory) {
+                return listFactory();
+            }],
+            // equivalent of `ctrl.images`
+            compactResults$: ['results', function(results) {
+                return results.items$.
+                    map(items => items.filter(angular.identity)).
+                    shareReplay(1);
+            }],
+            // set of selected images resources
+            selectedImages$: ['selection', 'compactResults$', function(selection, compactResults$) {
```

Comment:
Yes, the preview-selection service is now [delete](https://github.com/guardian/grid/pull/1200/files#diff-533b4c2674310e7ac52f494543d3e8a0L1). One of the differences is that previously the service was a singleton that lives across the whole lifecycle of the app, and therefore needs to be manually [purged](https://github.com/guardian/grid/pull/1200/files#diff-b8c6d4770b5b04294cc8c88d27ae65d3L422) when leaving the search view that uses it. Now the selection is instantiated as part of the search state. Less global state.